packages/@sanity/form-builder/src/inputs/DateInputs/base/calendar/Calendar.tsx

```diff
@@ -151,6 +160,18 @@ export const Calendar = forwardRef(function Calendar(
     }
   }, [ref, focusCurrentWeekDay, focusedDate])
 
+  useEffect(() => {
```

Comment:
Could we add a comment explaining in rough terms what this `useEffect` call is intended to handle? It's not obvious to me reading the code